health/health.d/netfilter.conf

```diff
@@ -15,5 +15,8 @@ component: Network
      warn: $this > (($status >= $WARNING)  ? (85) : (90))
      crit: $this > (($status == $CRITICAL) ? (90) : (95))
     delay: down 5m multiplier 1.5 max 1h
-     info: netfilter connection tracker table size utilization
+     info: Percentage of used Netfilter connections. \
```

Comment:
The project [Netfilter](https://www.netfilter.org/about.html#history) names the project in lower case, but the Core Team in uppercase, I think here we could use in lowercase to keep patter with official documentation.